<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videogameClass</class>
 <widget class="QMainWindow" name="videogameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>636</width>
    <height>602</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>636</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Видеоигры</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="List">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>311</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbgame">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>296</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>Игра</string>
    </property>
    <widget class="QGroupBox" name="gpplatform">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>72</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Платформа</string>
     </property>
     <widget class="QRadioButton" name="rbtnX1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>35</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnPS4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PS4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnPC">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>36</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PC</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gpspecs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>176</y>
       <width>281</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>Характеристики</string>
     </property>
     <widget class="QCheckBox" name="flagmp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Мультиплеер</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="flagcrossplatform">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Кроссплатформа</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="flagsplit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Сплит-скрин</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_specs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>106</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Подробное описание</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="otherspecs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>261</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="flagavailability">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>119</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Имеется в наличии</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>24</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="title">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>44</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="locale">
      <locale language="Russian" country="Russia"/>
     </property>
     <property name="maxLength">
      <number>30</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>440</y>
       <width>26</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Цена</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="price">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>460</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="create"/>
    <addaction name="open"/>
    <addaction name="save"/>
    <addaction name="saveas"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="create">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="saveas">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
